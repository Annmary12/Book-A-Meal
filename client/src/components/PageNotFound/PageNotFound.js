import React from 'react';
import styles from './pagenotfound.scss';
import Layout from '../Layout/Layout.js';

export default () => (
  <Layout>
    <div className={styles.background}>
      <div className={styles.notFound}>
          404<br />
          Page not found
      </div>
    </div>
  </Layout>
);
