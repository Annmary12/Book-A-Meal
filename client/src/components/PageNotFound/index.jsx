import React,{ Fragment } from 'react';
import NavBar from '../shared/NavBar/index.jsx';

export default () => (
  <Fragment>
    <NavBar />
    <div>Page not found</div>
  </Fragment>
);
