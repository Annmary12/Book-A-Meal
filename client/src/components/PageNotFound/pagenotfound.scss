.background {
  width: 100%;
  height: 100vh;
  background-image:
    repeating-linear-gradient(
      -45deg,
      transparent,
      transparent 1rem,
      #ccc 1rem,
      #ccc 2rem
    );
  background-size: 200% 200%;
  animation: barberpole 200s linear infinite;
}

@keyframes barberpole {
  100% {
    background-position: 100% 100%;
  }
}

.notFound {
  background-color: white;
  border-radius: 5px;
  margin: auto;
  display: flex;
  flex-direction: column;
  margin-bottom: 20px;

  box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);

  @media only screen and (max-width: 600px) {
    padding-top: 120px;
    width: 70%;
  }


  @media only screen and (min-width: 600px) {
    width: 50%;
  }

  @media only screen and (min-width: 768px) {
    width: 50%;
  }

  @media only screen and (min-width: 992px) {
    width: 50%;
  }

  text-align: center;
  padding-top: 17px;
  padding-bottom: 30px;
  font-size: 40px;
}