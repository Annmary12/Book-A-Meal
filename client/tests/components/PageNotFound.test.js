import React from 'react';
import PageNotFound from '../../src/components/PageNotFound/PageNotFound';

describe('PageNotFound component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<PageNotFound />);

    expect(wrapper).toMatchSnapshot();
  })
})