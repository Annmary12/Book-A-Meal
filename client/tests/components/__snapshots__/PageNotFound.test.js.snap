// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component should match snapshot 1`] = `
<div
  className="background"
>
  <div
    className="notFound"
  >
    <img
      alt="not found"
      className="image"
      src="test-file-stub"
      width="150"
    />
    <p>
      Page not found
    </p>
  </div>
</div>
`;
